### PR TITLE
buildbot-workers: add akira-home worker

### PR DESCRIPTION
**Pull request by Akira25**
Adds my personal worker in code. Deployed already.

### CODE REVIEW

File: inventory/group_vars/all [hunk 1]
```diff
@@ -1,79 +1,87 @@
 $ANSIBLE_VAULT;1.1;AES256
```
Comment (Noki) [inline inventory/group_vars/all:1]: Yaml-Lint will fail without `---` at the beginning

File: inventory/host_vars/buildbot-worker-akira/main.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+$ANSIBLE_VAULT;1.1;AES256
```
Comment (Noki) [inline inventory/host_vars/buildbot-worker-akira/main.yml:1]: Yaml-Lint will fail without `---` at the beginning

**Review by Noki — CHANGES_REQUESTED** [review]

File: inventory/group_vars/all (hunk 1 above)
Comment (Akira25) [reply inventory/group_vars/all:1]: The --- was encrypted by ansible-vault. Adding another one prevents decryption of the file. Maybe we can fix this by excluding encrypted files from the YAML-Linter?

Comment (Noki) [reply inventory/group_vars/all:1]: I changed the pull request for the yaml-lint workflow to ignore the ansible vault files (https://github.com/freifunk-berlin/ansible/pull/61) and added another pull request to revert the changes in other ansible vault files (https://github.com/freifunk-berlin/ansible/pull/64)

**Review by Noki — APPROVED** [review]
looks good to me

**Comment by nicolasberens** [comment]
i am not happy with having all the stuff in `inventory/group_vars/all` since these vars are only useful for the buildbotworker and the buildbot group.

On the other hand, i am absolutely fine with merging this right now, since this would just be nitpicking ;-)

(we might need to have a chat about the structure  of this reposity anyway)

**Comment by Akira25** [comment]
Yep, I am open for other ways, as I also feel, that having these vars in `all` is a bit counter-intuitive.

**Pull request merged**